content/docs/1.6.0/references/settings.md

```diff
@@ -470,7 +470,7 @@ Number of millicpus on each node to be reserved for each instance manager pod wh
 
 > **Warning:**
 >  - Specifying a value of 0 disables CPU requests for instance manager pods. You must specify an integer between 1000 and 8000. 
->  - This is a global setting. Modifying the value triggers an automatic restart of the instance manager pods. Do not modify the value while volumes are still attached.
+>  - This is a global setting. Modifying the value triggers an automatic restart of the Instance Manager pods. However, V2 Instance Manager pods that use this setting are restarted only when no instances are running.
```

Comment:
Why only v2?

Reply:
Because this is the section for `Guaranteed Instance Manager CPU for V2 Data Engine`
Here is for V1 https://github.com/longhorn/website/pull/858/files#diff-fa0cc835e2d54b4934afb441d3f5124d01daa7556cf3d1c2037ee1bbd4462d06R876